<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detector-SAM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>File Control Panel</string>
    </property>
    <widget class="QLabel" name="label_current_folder">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Open Yet</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_detect_contour">
     <property name="geometry">
      <rect>
       <x>399</x>
       <y>66</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect Contour</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_goto_previous">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_goto_next">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>CurrentFolder:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_file">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CurrentImage:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Arguments Control</string>
    </property>
    <widget class="QLabel" name="label_scale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ShowScale:0.25</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_scale">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_actual_scale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ActualScale (um/pix): Not Set Yet</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Contour Control</string>
    </property>
    <widget class="QLabel" name="label_minmaxArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>MinArea:   , MaxArea:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_delete_edge_contour">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>211</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Edge Contours</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_area">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>321</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_draw_contours">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw Contour</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>UoM 10652989</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFolder"/>
    <addaction name="separator"/>
    <addaction name="actionExportData"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionModifyMode"/>
    <addaction name="separator"/>
    <addaction name="actionMeasureScale"/>
    <addaction name="separator"/>
    <addaction name="actionImportScale"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFolder"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionModifyMode"/>
   <addaction name="separator"/>
   <addaction name="actionMeasureScale"/>
   <addaction name="actionImportScale"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionExportData"/>
  </widget>
  <action name="actionOpenFolder">
   <property name="text">
    <string>OpenFolder</string>
   </property>
  </action>
  <action name="actionExportData">
   <property name="text">
    <string>ExportData</string>
   </property>
  </action>
  <action name="actionModifyMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>EditMode</string>
   </property>
  </action>
  <action name="actionMeasureScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MeasureScale</string>
   </property>
  </action>
  <action name="actionImportScale">
   <property name="text">
    <string>ImportScale</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
